### PR TITLE
/forecasts route

### PR DESCRIPTION
**Pull request by acron0**
This adds the `/forecasts` GET route into a Liberator resource.

### CODE REVIEW

File: src/witan/app/forecast.clj [hunk 1]
```diff
@@ -3,30 +3,65 @@
...
+(defn get-forecasts
+  []
+  (c/exec (hayt/select :forecast_headers)))
+
+;;;;;;
+
+(defresource forecasts
+  :allowed-methods #{:get :post}
+  :available-media-types ["application/json"]
+  :handle-ok (fn [_] (s/validate
```
Comment (elisehuard) [inline src/witan/app/forecast.clj]: I remember Bruce saying that schema validation more part of handle-malformed (correct me if wrong)

File: test/witan/app/handler_test.clj [hunk 2]
```diff
@@ -4,10 +4,42 @@
...
+     :name "My Forecast 3",
+     :descendant_id nil,
+     :created #inst "2015-10-06T12:44:17.240-00:00",
+     :series_id #uuid "197481d6-df2a-4175-a288-d596a9709322",
+     :in_progress false,
+     :id #uuid "7185c4e4-739e-4eb8-8e37-f3f4b618ac1d",
+     :version 0,
+     :owner #uuid "cac4ba3a-07c8-4e79-9ae0-d97317bb0d45"}))
+
+(defn auth-header [token] {"Authorization" (str "Token " token)})
```
Comment (elisehuard) [inline test/witan/app/handler_test.clj]: nice one, was thinking the same. maybe we can abstract it even more later (between all routes that have to be authenticated and not, like buddy examples)

File: src/witan/app/forecast.clj (hunk 1 above)
Comment (acron0) [inline src/witan/app/forecast.clj]: IIUC `handle-malformed` is called when the request is found to be malformed (from the `malformed?` key). It's not used to check outward results, just incoming stuff.

Comment (elisehuard) [inline src/witan/app/forecast.clj]: you are right

**Comment by elisehuard** [comment]
lgtm :shipit:

**Pull request merged**